queries.sql

```diff
@@ -13,3 +13,55 @@ SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+  UPDATE animals
+  SET species = 'unspecified';
+
+ROLLBACK;
+
+BEGIN;
+
+  UPDATE animals 
+  SET species = 'digimon'
+  WHERE name LIKE '%mon%';
+
+  UPDATE animals
+  SET species = 'pokemon'
+  WHERE species is NULL;
+
+COMMIT;
+
+BEGIN;
+  TRUNCATE TABLE animals;
+ROLLBACK;
+
```

Comment:
- [ ] Again here kindly add a select statement  to check if changes were made after the `truncate` statement and after the `rollback` statement.